### PR TITLE
Map displays area searched, even if no results found

### PR DESCRIPTION
**Pull request by brangi**
## Description
issue: https://github.com/department-of-veterans-affairs/va.gov-team/issues/16945

## Screenshots

Screen Shot 2020-12-29 at 12 50 29 PM

## Acceptance Criteria

- [x] map displays the area entered, even if no results are found.
- [x] when no results are found, after zooming out or panning the map, the "search this area" button appears
- [x] clicking the "search this area" button sends a radius determined by the new map bounding box
- [x] new messaging is displayed when no results found
- [x] unit tests are added to cover new code
- [x] no new bugs are introduced

### CODE REVIEW

File: src/applications/facility-locator/containers/FacilitesMap.jsx [hunk 1]
```diff
@@ -585,6 +585,30 @@ const FacilitiesMap = props => {
...
+        props.results.length === 0 &&
+        map &&
+        props.currentQuery.searchCoords
+      ) {
+        const { searchCoords } = props.currentQuery;
+        const locationBounds = new mapboxgl.LngLatBounds();
+        const markerElement = buildMarker('currentPos');
+        new mapboxgl.Marker(markerElement)
+          .setLngLat([searchCoords.lng, searchCoords.lat])
+          .addTo(map);
```
Comment (mpelzsherman) [inline src/applications/facility-locator/containers/FacilitesMap.jsx:602]: These 3 lines are repeated 3 times in this file. Consider moving into a helper function?

**Comment by mpelzsherman** [comment]
After zooming to the selected location and finding no results, if the Veteran zooms out, we should display the “Search this area” button.

**Comment by mpelzsherman** [comment]
[2020-12-23T05:14:33.624Z] [0m  Mobile[0m
[2020-12-23T05:14:36.140Z]   [31m  1) should render in mobile layouts and tabs actions work[0m
[2020-12-23T05:14:39.410Z]   [32m  ✓[0m[90m should render the appropriate elements at each breakpoint[0m[31m (3789ms)[0m
[2020-12-23T05:14:39.410Z] 
[2020-12-23T05:14:39.410Z] 
[2020-12-23T05:14:39.410Z] [92m [0m[32m 1 passing[0m[90m (6s)[0m
[2020-12-23T05:14:39.410Z] [31m  1 failing[0m
[2020-12-23T05:14:39.410Z] 
[2020-12-23T05:14:39.410Z] [0m  1) Mobile
[2020-12-23T05:14:39.410Z]        should render in mobile layouts and tabs actions work:
[2020-12-23T05:14:39.410Z] [0m[31m     CypressError: `cy.type()` failed because it targeted a disabled element.
[2020-12-23T05:14:39.410Z] 
[2020-12-23T05:14:39.410Z] The element typed into was:
[2020-12-23T05:14:39.410Z] 
[2020-12-23T05:14:39.410Z]   > <input id="street-city-state-zip" name="street-city-state-zip" type="text" title="Your location: Street, City, State or Postal code" required="" value="" style="font-weight: bold;">
[2020-12-23T05:14:39.410Z] 
[2020-12-23T05:14:39.410Z] Ensure the element does not have an attribute named `disabled` before typing into it.

**Comment by mpelzsherman** [comment]
[2020-12-23T05:15:19.046Z] [0m  1) Facility search
[2020-12-23T05:15:19.046Z]        should render breadcrumbs :
[2020-12-23T05:15:19.046Z] [0m[31m     AssertionError: Timed out retrying: Expected to find element: `.all-details`, but never found it.[0m[90m
[2020-12-23T05:15:19.046Z]       at Context.eval (http://vets-website:3001/__cypress/tests?p=src/applications/facility-locator/tests/e2e/search.cypress.spec.js:674:30)
[2020-12-23T05:15:19.046Z] [0m

**Comment by mmiddaugh** [comment]
https://www.loom.com/share/f6e5c86a07804e039d379fad1cf4fc0f
Good improvements! Noticed this odd behavior - the button displays longer as I zoom out compared to when I zoom back in

**Review by mpelzsherman — APPROVED** [review]

**Pull request merged**